packages/x-charts-pro/src/internals/plugins/useChartProExport/useChartProExport.types.ts

```diff
@@ -12,31 +12,34 @@ export interface UseChartProExportState {
   export: {};
 }
 
-/**
- * The options to apply on the Print export.
- * @demos
- *   - [Print export](https://mui.com/x/react-charts/export/#print-export-as-pdf)
- */
-export interface ChartPrintExportOptions {
+export interface ChartExportOptions {
   /**
    * The name of the file without the extension.
    * @default The title of the document the chart belongs to
    */
   fileName?: string;
+  /**
+   * Callback function that is called before the export is triggered.
+   * It can be used to modify the iframe or perform any other actions before the export, such as updating styles,
+   * removing elements, etc.
+   * @param {HTMLIFrameElement} iframe containing the chart to be exported.
+   */
+  onBeforeExport?: (iframe: HTMLIFrameElement) => void;
```

Comment:
I believe this should also accept async functions.
```suggestion
  onBeforeExport?: (iframe: HTMLIFrameElement) => Promise<void> | void;
```